tests/cartesian_tests/unit_tests/test_exec_info.py

```diff
@@ -69,9 +70,18 @@ def diffusion_def(
                 flux_x[0, 0, 0] - flux_x[-1, 0, 0] + flux_y[0, 0, 0] - flux_y[0, -1, 0]
             )
 
-    def compile_stencils(self, backend):
-        self.advection = gtscript.stencil(backend=backend, definition=self.advection_def)
-        self.diffusion = gtscript.stencil(backend=backend, definition=self.diffusion_def)
+    def compile_stencils(self, backend, worker_id):
+        modulename = inspect.currentframe().f_back.f_globals["__name__"]
```

Comment:
Just a question: is this really needed? I guess any name could be used as prefix, isnt'it?

Reply:
Sure. This gives some additional guarantee that it is unique. It is also exactly what the default is if you don't provide `name` (It's a copy paste from there.)